src/plugins/formulas/hyperformulaSetup.js

```diff
@@ -0,0 +1,74 @@
+import staticRegister from '../../utils/staticRegister';
+import hyperformulaDefaultSettings from './hfDefaultSettings';
+
+/**
+ * Registers HyperFormula as a global entity and applies hooks allowing a multi-sheet setup.
+ *
```

Comment:
To make the "shared HF instance" scenario working - in that case, every HOT instance will read the HF instance from the global registry.
But when I think about it, you have a point there - if we have to pass the HF instance in the settings anyway, this might be redundant 🤔 This is a leftover from a previous config idea, where we would only pass the HF class, and the "multi-instance" thing was enabled by default, which is no longer the case.